src/cpp/include/openvino/genai/tokenizer.hpp

```diff
@@ -178,6 +179,7 @@ class OPENVINO_GENAI_EXPORTS Tokenizer {
     * @param add_special_tokens whether to add special tokens
     * @param max_length optional maximum length to which output will be truncated and/or padded. If not defined, taken from IR.
     * @param pad_to_max_length either pad to max_length, or pad to the longest sequence in the batch. Default is false.
+    * @param padding_side side to pad, either "left" or "right". If not defined value is taken from IR.
```

Comment:
Define what if there's no value in IR?
Add python docstrings

Reply:
Done. 
- The attribute is always present in the op. Default value into IR is written from the original HF/gguf model. Written that in the docstring.
- Added python docstring. It looks like this now
<img width="988" height="664" alt="image" src="https://github.com/user-attachments/assets/86c0b667-0ffa-4ed9-935c-d1909aff87f3" />

Reply:
What happens if it's not written? You don't fully control IRs because users are free to modify it